### PR TITLE
louis.is-a.dev

### PR DESCRIPTION
**Pull request by louloudev59**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/louis.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A portofolio",
+    "repo": "https://github.com/louloudev59/portofolio",
+    "owner": {
+        "username": "<louloudev59>",
+        "email": "louis@gmail.com",
+        "twitter": "<no-twitter>"
+    },
+    "record": {s
```
Comment (DEV-DIBSTER) [inline domains/louis.json:9]: Random S should be removed.

File: domains/louis.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A portofolio",
+    "repo": "https://github.com/louloudev59/portofolio",
+    "owner": {
+        "username": "<louloudev59>",
+        "email": "louis@gmail.com",
+        "twitter": "<no-twitter>"
```
Comment (DEV-DIBSTER) [inline domains/louis.json:7]: You can remove this line in that case.

File: domains/louis.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A portofolio",
+    "repo": "https://github.com/louloudev59/portofolio",
+    "owner": {
+        "username": "<louloudev59>",
```
Comment (DEV-DIBSTER) [inline domains/louis.json:5]: Username should not contain the <>, that's just to show it is a required property.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/louis.json (hunk 2 above)
Comment (louloudev59) [reply domains/louis.json:7]: The twitter line ?

File: domains/louis.json (hunk 3 above)
Comment (louloudev59) [reply domains/louis.json:5]: Oh ok, thanks

**Pull request closed without merging**